[Bug]: Spotless Precommit Fails for :sdks:java:io:kafka:compileTestJava

### What happened?

Spotless Precommit Fails for :sdks:java:io:kafka:compileTestJava:

```
:sdks:java:io:kafka:compileTestJava FAILED	
/runner/_work/beam/beam/sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProviderTest.java:253: error: cannot find symbol	
        () -> transform.expand(PCollectionRowTuple.empty(Pipeline.create())));	
                               ^	
  symbol:   variable PCollectionRowTuple	
  location: class KafkaReadSchemaTransformProviderTest	
/runner/_work/beam/beam/sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProviderTest.java:298: error: cannot find symbol	
        () -> transform.expand(PCollectionRowTuple.empty(Pipeline.create())));	
                               ^	
  symbol:   variable PCollectionRowTuple	
  location: class KafkaReadSchemaTransformProviderTest	
Note: Some input files use or override a deprecated API.	
Note: Recompile with -Xlint:deprecation for details.	
Note: Some input files use unchecked or unsafe operations.	
Note: Recompile with -Xlint:unchecked for details.	
2 errors
```

https://ge.apache.org/s/i2qjrxoudsvfy/console-log/task/:sdks:java:io:kafka:compileTestJava?anchor=15&page=1

### Issue Priority

Priority: 2 (default / most bugs should be filed as P2)

### Issue Components

- [ ] Component: Python SDK
- [X] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [X] Component: IO connector
- [ ] Component: Beam YAML
- [ ] Component: Beam examples
- [ ] Component: Beam playground
- [ ] Component: Beam katas
- [ ] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

--- a/sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProviderTest.java
+++ b/sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProviderTest.java
@@ -38,6 +38,7 @@ import org.apache.beam.sdk.schemas.transforms.SchemaTransform;
 import org.apache.beam.sdk.schemas.transforms.SchemaTransformProvider;
 import org.apache.beam.sdk.schemas.utils.YamlUtils;
 import org.apache.beam.sdk.values.PBegin;
+import org.apache.beam.sdk.values.PCollectionRowTuple;
 import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.collect.Lists;
 import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.collect.Sets;
 import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.io.ByteStreams;

